dataCompareR/tests/testthat/testLargeDataWarning.R

```diff
@@ -0,0 +1,35 @@
+# SPDX-Copyright: Copyright (c) Capital One Services, LLC 
+# SPDX-License-Identifier: Apache-2.0 
+# Copyright 2017 Capital One Services, LLC 
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); 
+# you may not use this file except in compliance with the License. 
+#
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 
+#
+# Unless required by applicable law or agreed to in writing, software distributed 
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied.
+
+#
+# UNIT TEST*: checks large data warning
+
+context("checks large data warning in rCompare")
+
+test_that("Silent for small data", {
+  
+  expect_silent(warnLargeData(iris,iris))
+  expect_silent(warnLargeData(iris,pressure))
+  expect_silent(warnLargeData(pressure,pressure))
+})
+
+
+test_that("Warns for large data", {
```

Comment:
One solution to the problem of unit testing is to make the function that does the check only take in the actual column and row counts, rather than the data frames. This of course means we still won't have a proper automated check for big data sets, but at least we could capture the part where it is sending a message for large values.

Outside of that, if we're going to have a unit test that Travis doesn't run within this file, then we should just comment out the entire `test_that` block, rather than putting in a meaningless placeholder like `expect_true(1==1)`.